### ISSUE TITLE
Posibles Logos

### ISSUE THREAD

**Issue by Pand-o-Ra** [javascript-beer/javascript.beer#1]
Buenas,
os paso algunas ideas para los logos ;)
Nota. Están enormes y son sólo bocetos, ok?

![logo_1](https://user-images.githubusercontent.com/78531756/107124727-eec96280-68a5-11eb-86e3-6c5b6941d6ab.png)
![logo_3](https://user-images.githubusercontent.com/78531756/107124729-effa8f80-68a5-11eb-983f-c8b2a9302eb0.png)
![logo_4](https://user-images.githubusercontent.com/78531756/107124730-effa8f80-68a5-11eb-8e49-49fdb661377a.png)
![logo_nuw_blanc](https://user-images.githubusercontent.com/78531756/107124731-f0932600-68a5-11eb-9446-a5b3df189cf4.png)
![logo_nuw_grey](https://user-images.githubusercontent.com/78531756/107124732-f0932600-68a5-11eb-9b37-7d3ebb4dc9dc.png)

**Comment by alexfernandez** [comment]
Me encanta el penúltimo :heart_eyes:

**Comment by alo** [comment]
> Me encanta el penúltimo 😍

¡A mí también! Original 🤣

**Comment by felixzapata** [comment]
entiendo que el último es la versión transparente del penúltimo

**Comment by Pand-o-Ra** [comment]
> 
> 
> entiendo que el último es la versión transparente del penúltimo

No. Son 2 diferentes. Uno con fondo blanco (limpio y estándar), y el otro con fondo gris. Este año, los colores que más se llevan este año son la [combinación de gris y amarillo](https://fashionunited.es/noticias/moda/pantone-desvela-el-color-del-ano-2021-una-combinacion-en-amarillo-y-con-el-numero-13/2020121034148)... (he hecho un guiño a las tendencias en colores).

**Comment by manufosela** [comment]
1, 4 y 2 son los que mas me gustan. En ese orden
